client/components/gsuite/gsuite-price/index.jsx

```diff
@@ -15,60 +15,42 @@ import { getAnnualPrice, getMonthlyPrice } from 'lib/gsuite';
  */
 import './style.scss';
 
-const GSuitePrice = ( { cost, currencyCode, showMonthlyPrice } ) => {
+const GSuitePrice = ( { currencyCode, product } ) => {
 	const translate = useTranslate();
 
+	const cost = product.cost || null;
```

Comment:
I did actually see the screen blow up a few times because product was null. Is there a way to mitigate this? Perhaps the product isn't loaded at the time.

<img width="1399" alt="Screenshot 2019-11-22 at 12 22 43 PM" src="https://user-images.githubusercontent.com/277661/69422068-0c52f080-0d23-11ea-97b3-d711b15285b5.png">

Reply:
Good catch, let me fix that.